draft-ietf-quic-transport.md

```diff
@@ -2686,9 +2709,9 @@ might also occur because the client's access network used a different NAT
 binding for the server's preferred address.
 
 Servers SHOULD initiate path validation to the client's new address upon
```

Comment:
This entire paragraph doesn't seem very specific to "Interaction of Client Migration and Preferred Address"

I think it'd be better to reference the section and not put these normative statements here, since they are so general.

Reply:
Agreed.

Reply:
Follow-up Q: Is this SHOULD specific to "Interaction of Client Migration and Preferred Address" or more general?  It seems more general, but I don't have a strong opinion.